### PR TITLE
Update servicing-stack-updates.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a related article - Windows server OS SSU catalog

per issue https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10905

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: windows/deployment/update/servicing-stack-updates.md [hunk 1]
```diff
@@ -59,3 +59,8 @@ Typically, the improvements are reliability and performance improvements that do
 ## Simplifying on-premises deployment of servicing stack updates
 
 With the Windows Update experience, servicing stack updates and cumulative updates are deployed together to the device. The update stack automatically orchestrates the installation, so both are applied correctly. Starting in February 2021, the cumulative update will include the latest servicing stack updates, to provide a single cumulative update payload to both Windows Server Update Services (WSUS) and Microsoft Catalog. If you use an endpoint management tool backed by WSUS, such as Configuration Manager, you will only have to select and deploy the monthly cumulative update. The latest servicing stack updates will automatically be applied correctly. Release notes and file information for cumulative updates, including those related to the servicing stack, will be in a single KB article. The combined monthly cumulative update will be available on Windows 10, version 2004 and later starting with the 2021 2C release, KB4601382.
+
+## Related Articles
+
+[Microsoft Servicing Stack Updates catalog for windows server operating system](https://www.catalog.update.microsoft.com/Search.aspx?q=Servicing%20Stack%20Update)
```
Comment (JohanFreelancer9) [inline windows/deployment/update/servicing-stack-updates.md:65]:
```suggestion
[Microsoft Servicing Stack Updates Catalog for Windows Server](https://www.catalog.update.microsoft.com/Search.aspx?q=Servicing%20Stack%20Update)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@aczechowski Please merge this PR. Thanks

**Review by aczechowski — CHANGES_REQUESTED** [review]
Thanks @msbemba for the edit... 
🤔 That link might be useful, but it does return almost 800 results. I suppose there's no harm in keeping it.
But I think the simpler way to address the issue is to just add "Windows Server" to the bullets under "Applies to" at the top of the article. (FYI, we are working on a standard way to note the applicability of an article, so that might change in the future, but for now that will suffice.)

**Comment by msbemba** [comment]
> Thanks @msbemba for the edit... 🤔 That link might be useful, but it does return almost 800 results. I suppose there's no harm in keeping it. But I think the simpler way to address the issue is to just add "Windows Server" to the bullets under "Applies to" at the top of the article. (FYI, we are working on a standard way to note the applicability of an article, so that might change in the future, but for now that will suffice.)

@aczechowski That's a good idea. I have made the changes as per your recommendation.

Thanks
Sri

**Review by aczechowski — APPROVED** [review]

**Pull request merged**